docker/manila/manila-share/Dockerfile.j2

```diff
@@ -9,24 +9,6 @@ LABEL maintainer="{{ maintainer }}" name="{{ image_name }}" build-date="{{ build
 
 {{ macros.enable_extra_repos(['ceph']) }}
 
-{% if base_package_type == 'rpm' %}
-    {% set manila_share_packages = [
-        'ceph-common',
-        'glusterfs-fuse',
-        'sqlite',
-    ] %}
-{% elif base_package_type == 'deb' %}
-    {% set manila_share_packages = [
-        'ceph-common',
-        'glusterfs-client',
-        'python3-cephfs',
-        'python3-rados',
-        'python3-rbd',
-        'sqlite3'
-    ] %}
-
-{% endif %}
-
```

Comment:
I suspect we need these.

the manila-share dockerfile already had the upstream "drop binary parts" [commit applied here.](https://github.com/ChameleonCloud/kolla/commit/1749da2fbf6b395b1a2cd77f349e0bea5d2f4a5e#diff-d3995d32e666aa0cf7f1c02c2351ad4fbcb8ef7fd2bac2284a77277a1ddfd90c)